_config/linktracking.yml

```diff
@@ -4,3 +4,12 @@ Name: cmslinktracking
 SilverStripe\ORM\DataObject:
   extensions:
     - SilverStripe\CMS\Model\SiteTreeLinkTracking
+---
+Name: cmslinktrackingvisibility
+---
+SilverStripe\CMS\Model\SiteTree:
+  show_link_tracking: true
+SilverStripe\Assets\File:
+  show_link_tracking: true
+SilverStripe\Security\Member:
+  show_link_tracking: true
```

Comment:
What’s the use-case for this on `Member`? I’m drawing a blank 😬

Reply:
I suspect it's being set to true to mirror current behaviour?